packages/mobile/metro.config.js

```diff
@@ -63,6 +63,7 @@ module.exports = (async () => {
         ...require('node-libs-react-native'),
         // Alias for 'src' to allow for absolute paths
         app: path.resolve(__dirname, 'src'),
+        '@audius/harmony-native': path.resolve(__dirname, 'src/harmony-native'),
```

Comment:
We could eventually pull harmony-native into it's own package too, prob not necessary tho